pkg/controller/controller.go

```diff
@@ -737,10 +745,18 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) {
 			}
 		}
 
+		ns, _ := c.kubeClient.Core().Namespaces().Get(instance.Namespace)
```

Comment:
Oopsie, pushed the code where I was testing the negative case.

Fixed, thanks!